eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -654,7 +654,7 @@ object Helpers {
       def firstClosingFee(commitments: Commitments, localScriptPubkey: ByteVector, remoteScriptPubkey: ByteVector, feerates: ClosingFeerates)(implicit log: LoggingAdapter): ClosingFees = {
         import commitments._
         // this is just to estimate the weight, it depends on size of the pubkey scripts
-        val actualLocalScript = if (channelFeatures.hasFeature(Features.UpfrontShutdownScript)) localParams.defaultFinalScriptPubKey else localScriptPubkey
+        val actualLocalScript = if (channelFeatures.hasFeature(Features.UpfrontShutdownScript)) localParams.defaultFinalScriptPubKey.get else localScriptPubkey
```

Comment:
We don't need to constantly refer to the feature:
```suggestion
        val actualLocalScript = localParams.defaultFinalScriptPubKey_opt.getOrElse(localScriptPubkey)
```

Both scripts were validated earlier in the code.

Reply:
done in https://github.com/ACINQ/eclair/pull/2565/commits/21a46ca164a11a5954b87d4569fc77b30954ef0a